### PR TITLE
bokehjs fix for height calc

### PR DESCRIPTION
**Pull request by Hanterdro**
This pull Requests fix incorrect height calcs

See: https://github.com/bokeh/bokeh/issues/5716

### CODE REVIEW

File: bokehjs/src/coffee/models/layouts/widget_box.coffee [hunk 1]
```diff
@@ -58,8 +58,8 @@ export class WidgetBoxView extends LayoutDOMView
     height = 0
     # We have to add on 10px because widgets have a margin at the top.
     for own key, child_view of @child_views
-      height += child_view.el.scrollHeight
-    return height + 20
+      height += $(child_view.el).outerHeight(true)
```
Comment (mattpap) [inline bokehjs/src/coffee/models/layouts/widget_box.coffee]: We don't use jquery anymore. This has to be implemented with native DOM functionality.

**Comment by Hanterdro** [comment]
Okay... I replaced the jQuery stuff with vanillaJS.
I readded the magic `+10` for the height, but I don't think it's because of the margin of the elements. I think it's needed for the `widget-box` container, which has a padding-top and padding-bottom of 5px.

**Comment by bryevdv** [comment]
HI @Hanterdro thanks for looking in to this some! The layout system is a bit complicated, ultimately Bokeh uses a constraint solver to lay things out. It's possible the value is not getting fed into that pipeline. I will try to take a look myself using your branch to see if I can spot where the broken link might be, but it will be later this week.

**Comment by bryevdv** [comment]
@mattpap any thoughts on this? Should we just merge, or is it superseded by your other PR?

**Comment by bryevdv** [comment]
ping @mattpap for comment

**Comment by bryevdv** [comment]
@mattpap we should either merge or close this

**Comment by bryevdv** [comment]
Actually, looking at the failing tests, I think this would be non-trivial to fix up. I am going to go ahead and close. @Hanterdro thanks for the PR, I regret that we could not get it merged. Layout is a large effort and one of the more complicated areas, and much code has changed around this PR. I do hope you will consider contributing again.

**Pull request closed without merging**